perception/autoware_lidar_centerpoint/lib/preprocess/voxel_generator.cpp

```diff
@@ -51,7 +51,7 @@ bool VoxelGeneratorTemplate::enqueuePointCloud(
 
 std::size_t VoxelGenerator::generateSweepPoints(float * points_d, cudaStream_t stream)
 {
-  size_t point_counter = 0;
+  std::size_t point_counter = 0;
```

Comment:
Since this change is irrelevant from the core idea of this PR which may confuse other developers who will refer to this PR in the future, can we merge the following PR first?
https://github.com/autowarefoundation/autoware.universe/pull/8820

Reply:
Sorry I missed some `size_t` in the above PR :cry: 
Let's just merge this PR as it is.